airflow/plugins_manager.py

```diff
@@ -57,8 +57,21 @@ class AirflowPlugin:
     # The function should have the following signature:
     # def func_name(stat_name: str) -> str:
     stat_name_handler = None  # type: Any
+
+    # A list of global operator extra links that can redirect users to
+    # external systems. These extra links will be available on the
+    # task page in the form of buttons.
+    #
+    # Note: the global operator extra link can be overridden at each
+    # operator level.
     global_operator_extra_links = []  # type: List[BaseOperatorLink]
 
+    # If ``store_serialized_dags = True`` (DAG Serialization is turned on)
+    # Operator extra links have to be defined in plugins to work.
+    # These extra links will be available on the task page in form of
+    # buttons.
+    serialized_operator_extra_links = []  # type: List[BaseOperatorLink]
```

Comment:
I would just call this `operator_extra_links` -- that way we don't have to rename it if/when we make serialized the only way of working.

Reply:
Updated